### ISSUE TITLE
Requesting Standards Position for Mixed Content Level 2

### ISSUE THREAD

**Issue by mozfreddyb** [mozilla/standards-positions#668]
## Request for Mozilla Position on an Emerging Web Specification

* Specification Title: Mixed Content Level 2
* Specification or proposal URL: https://w3c.github.io/webappsec-mixed-content/
* Bugzilla URL: [bug 1779757](https://bugzilla.mozilla.org/show_bug.cgi?id=1779757)
* Mozillians who can provide input (optional): @dveditz

### Other information

We already implement Mixed Content 1. The newest bits are automated upgrading of subresources that are not already blocked by default (e.g., not scripts). The spec also introduces blocking of "mixed downloads", which AFAIU we also implement.

Implementing this would increase HTTPS adoption and would set us on a clearer path for potential future work in the realm of HTTPS-First/HTTPS-Only.

**Comment by martinthomson** [comment]
These are good things, maybe even important.  @mozfreddyb or @dveditz, do you think you can create a dashboard entry for this one?

**Comment by mozfreddyb** [comment]
PR is in #669
